kong/db/schema/entities/services.lua

```diff
@@ -50,23 +50,23 @@ return {
 
   entity_checks = {
     { conditional = { if_field = "protocol",
-                      if_match = { one_of = { "tcp", "tls", "udp", "grpc", "grpcs" }},
+                      if_match = { one_of = { "tcp", "tls", "udp", "grpc", "grpcs" } },
```

Comment:
There is one concern, Would you need to consider compatibility issues？ what I mean is downgrade compatibility issues, because we've modified our schema.

Reply:
Can you give me an example? 
From my perspective, the fields `tls_verify` and `ca_certificate` are optional.

Reply:
I mean if a customer from 3.5 -> 3.4, and 3.5 versions use this new option, 3.4 does not have these options. it may be the result of some schema error. We need to consider this schema error.  For example, you can follow this do some change https://github.com/Kong/kong/blob/master/kong/clustering/compat/checkers.lua#L118

Reply:
@oowl Please take a look at it again. Thx